test/JDBC/input/storedProcedures/BABEL-5059-vu-prepare.sql

```diff
@@ -0,0 +1,247 @@
+-- nvarchar
+CREATE PROCEDURE babel_5059_proc_test_1 (@a NVARCHAR, @b NVARCHAR OUTPUT) AS BEGIN SET @b=@a; SELECT @b; END;
```

Comment:
I only see tests for procedure, Please add tests for functions as well.

Reply:
Added for functions with all the datatypes that we are updating typmod value for, UDT testing ,  ITVF and MSTVF